.github/workflows/actions.yml

```diff
@@ -326,6 +326,8 @@ jobs:
       - name: install qt using aqtinstall
         uses: jurplel/install-qt-action@v2
         if: matrix.qt-version
+        env:
```

Comment:
This appears to be a new change, not a reversion.

Reply:
As I noted above

> I also cherry-picked CI commit that fixes the macOS legacy build.

This is a fix from #5627 - if we want to merge this into 3.12 and make another release, we need that fix for the macOS legacy build.

Reply:
OK, I've withdrawn that review. Thanks (and sorry that I missed / didn't understand the initial comment).